notebooks/11-test-verifications-to-get-human-labels.ipynb

```diff
@@ -0,0 +1,6704 @@
+{
```

Comment:
<p>got it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bleu/farcaster-social-graph/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>